src/test/integration/Dockerfile

```diff
@@ -0,0 +1,5 @@
+FROM python:2-onbuild
+
+CMD [ "python", "./integration_tests.py", "-v", "-i", "configuration.json" ]
+
+USER nobody:nogroup
```

Comment:
I'm trying to read the docs, and I can't tell if the CMD command you specified in line 3 will run as root or as the nobody user: https://docs.docker.com/engine/reference/builder/#user . It seems that the USER affects CMD instructions that follow it?